.circleci/config.yml

```diff
@@ -1320,23 +1314,17 @@ jobs:
       image: circleci/classic:201808-01
     steps:
       - checkout
-      - run:
-          name: ci job started
-          command: './scripts/ci-job-started.sh'
       - attach_workspace:
           at: .
       - run: docker info
       - run: docker run --privileged linuxkit/binfmt:v0.6
-      - run: cp dist/grafana-latest.linux-*.tar.gz packaging/docker
-      - run: cd packaging/docker && ./build.sh --fast "${CIRCLE_SHA1}"
       - run:
-          name: ci job failed
-          command: './scripts/ci-job-failed.sh'
-          when: on_fail
+          name: Copy Grafana archives
+          command: cp dist/grafana-latest.linux-*.tar.gz packaging/docker
       - run:
-          name: ci job succeeded
-          command: './scripts/ci-job-succeeded.sh'
-          when: on_success
+          name: Build Docker images
+          command: |
+            pushd packaging/docker && ./build.sh --fast "pr-${CIRCLE_SHA1}" && popd
```

Comment:
There's no need for pushing and popping the directory, because following steps won't be affected by the directory change in this step. At least I'm _pretty_ sure that's the case.

Reply:
this was old, synced with master and it does not look like this anymore